modules/machineconfig-modify-registry.adoc

```diff
@@ -102,7 +102,7 @@ Starting pod/ip-10-0-141-142us-east-2computeinternal-debug ...
 [source,terminal]
 ----
 sh-4.2# chroot /host
-sh-4.4# cat /etc/containers/registries.d/99-worker-unqualified-search-registries.conf
+sh-4.4# cat /etc/containers/registries.conf.d/99-worker-unqualified-search-registries.conf
 unqualified-search-registries = ['registry.access.redhat.com', 'docker.io', 'quay.io']
 sh-4.4# exit
 ----
```

Comment:
The update LGTM. Should these `sh-4.4#` commands be separated into individual code blocks?